OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -1690,21 +1553,20 @@ else if (keyValues.isNull(key) || "".equals(value)) {
             }
 
             if (!toSend.toString().equals("{}")) {
+               logger.debug("Available tags to send: " + toSend.toString());
```

Comment:
Can we add better logging here? The else statement has "Send tags ended successfully"

Thus, what is the opposite of that? Why did the send tags not send successfully?

Reply:
For context, I am falling into this codepath while testing RN and don't know why. Better logging here (and across all methods) would be helpful.

Logs:
```
2020-09-24 20:13:25.651 11163-11247/com.rnonesignal I/ReactNativeJS: Attempting to send tags
2020-09-24 20:13:25.672 11163-11248/com.rnonesignal D/OneSignal: Available tags to send: {"a":"1","b":"2"}
```